argo/kustomization.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- base/argo-clusterrole-binding.yaml
+- base/argo-clusterrole.yaml
+- base/argo-crd.yaml
+- base/argo-sa.yaml
+- base/argo-ui-clusterrole-binding.yaml
+- base/argo-ui-clusterrole.yaml
+- base/argo-ui-deployment.yaml
+- base/argo-ui-sa.yaml
+- base/argo-ui-service.yaml
+- base/argo-wfc-configmap.yaml
+- base/argo-wfc-deployment.yaml
+commonLabels:
+  kustomize.component: argo
+images:
+  - name: argoproj/argoui
+    newName: argoproj/argoui
+    newTag: v2.2.0
+  - name: argoproj/workflow-controller
+    newName: argoproj/workflow-controller
+    newTag: v2.2.0
+configMapGenerator:
+- name: workflow-controller-parameters
+  env: params.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
+- name: executorImage
+  objref:
```

Comment:
Why doesn't the user just modify the ConfigMap rather than setting all the parameters here?

Reply:
Filed #23